src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -346,6 +354,15 @@ export module DotNet {
         ? parseJsonWithRevivers(resultJsonOrExceptionMessage)
         : new Error(resultJsonOrExceptionMessage);
       completePendingCall(parseInt(asyncCallId), success, resultOrError);
+    },
+
+    /**
+     * Receives notification that a byte array is available for transfer.
```

Comment:
```suggestion
     * Receives notification that a byte array is being transferred from .NET to JS.
```

(same reason as above)